docs/site/Sequence.md

```diff
@@ -228,19 +270,53 @@ page [Parsing requests](Parsing-requests.md)
 
 ### Invoking controller methods
 
-{% include content/tbd.html %}
-
-- How to use `invoke()` in simple and advanced use cases.
-- Explain what happens when you call `invoke()`
-- Mention caching use case
-- Can I call invoke multiple times?
+The `invoke` sequence action simply takes the parsed request parameters from the
+`parseParams` action along with non-decorated arguments, calls the corresponding
+controller method or route handler method, and returns the value from it. The
+default implementation of
+[invoke](https://github.com/strongloop/loopback-next/blob/6bafa0774662991199090219913c3dc77ad5b149/packages/rest/src/providers/invoke-method.provider.ts)
+action calls the handler function for the route with the request specific
+context and the arguments for the function. It is important to note that
+controller methods use `invokeMethod` from `@loopback/context` and can be used
+with global and custom interceptors. See
+[Interceptor docs](https://github.com/strongloop/loopback-next/blob/d1646d085bbd5ac8df0d289fb3b8d5b5f5370386/docs/site/Interceptors.md#use-invokemethod-to-apply-interceptors)
```

Comment:
Can you link to the website instead (https://loopback.io/doc/en/lb4/Interceptors.html#use-invokemethod-to-apply-interceptors)?

Reply:
Good catch @nabdelgadir 👍 (I've also fixed the indentation on code blocks).